library-best-practices/JLBP-6.md

```diff
@@ -28,21 +28,22 @@ package (2).
 
 Recommendations:
 
-- When making a breaking change, take one of the two following approaches:
-  - *If* the new library surface is delivered under a new Java package, either
-    use a different Maven ID (different group ID or artifact ID) or bundle the
-    old and new packages together under the original Maven ID.
-  - *If* the breaking change is made in-place and the Java package is kept the
-    same, use the same Maven ID (same group ID and same artifact ID).
-  - There is a tradeoff between the two options above: if a library breaks the
-    surface for a method that is not used in any consumers, there would be no
-    diamond dependency conflicts due to the breakage, so a package rename is
-    a high-cost transition with no real benefit. Conversely, if a library breaks the
-    surface for a method that is used in many places at
-    multiple levels of the dependency graph, there would be a huge amount of
-    diamond dependency conflicts if the package isn't renamed, so a package
-    rename is preferrable. Generally, the wider the usage of the features
-    that need to break, the higher the value from renaming the Java package.
+- When making a breaking change, take one of the following approaches:
+  1. If the new library surface is delivered under a new Java package, either
```

Comment:
This should really be three approaches, but let's fix this later

Reply:
ok, keeping this for now.